### DISCUSSION TITLE
button dark / light

### DISCUSSION THREAD

**Discussion by wgnalvian** [community/community#32108] [Other Feature Feedback, Questions, & Ideas]
where is button dark / light ?? it disappeared

**Comment by radar-blip** [comment]
In your settings, under Appearance.

**Comment by ihnatenkoo** [comment]
**Hello, wgnalvian! See here :)**

![2022-09-06 13 32 56](https://user-images.githubusercontent.com/89586345/188613871-7530c57c-0076-4468-a1ce-471e97958d7a.png)
